forc-plugins/forc-doc/src/render.rs

```diff
@@ -463,7 +464,7 @@ impl Renderable for TyStorageDeclaration {
             attributes,
         } = &self;
         let name = "Contract Storage".to_string();
-        let code_span = span.as_str().to_string();
+        let code_span = parse::parse_format::<sway_ast::ItemStorage>(span.as_str());
```

Comment:
Hmm looks like `parse_format` returns the formatted code as a string rather than a span, if that is the case it might make sense to change variable names for readability.